lbuild/environment.py

```diff
@@ -65,6 +65,7 @@ def __init__(self, options, modules, module, outpath, buildlog):
             'options': self.options,
         }
 
+        self.log = None
```

Comment:
Why not make this the property (with the code which is currently in the module) and remove everything from the module. That should be much easier and doesn't require overwriting it before the build calls.